cloudbuild.yaml

```diff
@@ -0,0 +1,50 @@
+steps:
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'bb-worker-debian8/'
+    
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'bb-worker-debian9/'
+
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'bb-worker-debian10/'
+
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'bb-worker-ubuntu1604/'
+
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'bb-worker-ubuntu1804/'
+
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'bb-worker-ubuntu1904/'
+
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'zcashd/'
+
+  - name: gcr.io/cloud-builders/docker
+    args: 
+      - 'build'
+      - '.'
+    dir: 'zcash-insight-explorer/'
+
+name: This is dumb
```

Comment:
Reviews were requested on this PR but this last addition has me wondering if it's a work in progress or if it's intended to be merged